test/metabase/query_processor_test/date_time_zone_functions_test.clj

```diff
@@ -27,7 +27,7 @@
          (mt/formatted-rows [int])
          (map first))))
 
-(mt/defdataset times-mixed
+(mt/defdataset times-mixed-2
```

Comment:
Temporarily changing this to 2 because I added a new table to this dataset, and because bigquery and snowflake don't update the database if the same database exists.

I'll clean this up and find a way to add the new table to the existing `times-mixed` database once the PR is approved.

Reply:
I've felt this pain recently as well. We really need a good way to do something like `(mt/reset-dataset <driver> <dataset-name>)` to force reset them. The mechanism for this exists we just need to safely expose it.